weblate/trans/forms.py

```diff
@@ -871,6 +871,7 @@ class AutoForm(forms.Form):
         label=_("Automatic translation mode"),
         choices=[
             ("suggest", _("Add as suggestion")),
+            ("approved", _("Add as approved translation")),
```

Comment:
This will need additional logic in `__init__` to show approved choice only if the user has permissions for that. `__init__` will need to get user object as a parameter for that and then check the permissions via `user.has_perm("unit.review", obj)`.

Reply:
done with the latest commit.